tools/dep_updaters/update-cjs-module-lexer.sh

```diff
@@ -2,6 +2,14 @@
 set -e
 # Shell script to update cjs-module-lexer in the source tree to a specific version
 
+# update cjs_module_lexer_version.h
+update_cjs_module_lexer_version(){
+    # get cjs-module-lexer version
+    CJS_MODULE_LEXER_VERSION=$1
+    # update this version information in src/cjs_module_lexer_version.h
+    echo "#define CJS_MODULE_LEXER_VERSION \"$CJS_MODULE_LEXER_VERSION\"  // NOLINT(build/header_guard)" > "$BASE_DIR/src/cjs_module_lexer_version.h"
```

Comment:
Should we include the header guards like @debadree25 did in https://github.com/nodejs/node/pull/45621 https://github.com/nodejs/node/blob/7984af69a090dd6d1f60ffe7e194d5e69bce0c20/tools/dep_updaters/update-undici.sh#L39-L46 for consistency? I think that should also help you get rid of the `NOLINT` comments. Let's also do the same for the base64 files and update the header files accordingly.

Reply:
Sure.